src/ansys/units/quantity.py

```diff
@@ -176,41 +159,59 @@ def to(self, to_units: Union[ansunits.Unit, str]) -> "Quantity":
 
         return Quantity(value=new_value, units=to_units)
 
+    def __float__(self):
+        return self.si_value
+
+    def __array__(self):
+        if _array:
+            if isinstance(self.value, (float)):
+                return np.array([self.value])
```

Comment:
This specific part is interesting. Cannot remember if we discussed this feature, but I see that this is what pint does also. We should support it too.